src/pages/TabHome.tsx

```diff
@@ -83,6 +84,7 @@ const TabHome = (props: HomeProps) => {
   const [isInfoModal, setIsInfoModal] = useState(false);
   const [isWelcomeModal, setIsWelcomeModal] = useState(false);
   const [periodTodayAlert] = useIonAlert();
+  const [noSaveAlert] = useIonAlert();
```

Comment:
`cannotSaveAlert` look better in my opinion

Reply:
Renamed